Updated mns-payment.json

brysonjbest opened the pull request:
Adding mns-payment.json

Review: https://brms-simulator.apps.silver.devops.gov.bc.ca/rule/6670cb5679140169e141e80d?version=inReview&_=1741794098718

sbuhovac commented:
Cris and I did validation testing and here's our feedback.

- Input "Residing In Care Facility" as TRUE always results in ineligible output but that should not always be the case. If we select input "Facility is an Alcohol Or Drug Care Facility" as TRUE then logically we must select "Residing In Care Facility" as TRUE and those two combined should result in Eligible output.

-  If there is an amount in input "Receiving Diet Supplement" it will reduce the amount of Additional Nutritional Items by the amount of Diet. This doesn't match policy that says "Recipients who are in receipt of the nutritional items portion of the MNS are not eligible for any other nutrition-related supplement". If the intent of this is to be applied only in the first month of eligibility then this setup is correct for One Time payments but not for Monthly/Ongoing.

- Input “Verification of Need” is something that Health Assistance determines as part of adjudication process, and that is not in scope for BRE. MNS should be treated like PWD designation as both are adjudicated by Health Assistance.

@tekarrawilkinson @prchristenson @Iliana777

brysonjbest commented:
@sbuhovac @tekarrawilkinson @prchristenson @Iliana777 

> 
>     * Input "Residing In Care Facility" as TRUE always results in ineligible output but that should not always be the case. If we select input "Facility is an Alcohol Or Drug Care Facility" as TRUE then logically we must select "Residing In Care Facility" as TRUE and those two combined should result in Eligible output.
>

- Updated to match mapping. if 'Residing In Care Facility' is true, then the rule checks to see if 'Facility is an Alcohol Or Drug Care Facility' is true. If 'Facility is an Alcohol Or Drug Care Facility' is true, then the client is eligible. However, if it is false, or empty, the rule will default to ineligible, as the client is in a facility that has not been verified to be an alcohol or drug care facility. As well, if the input 'Residing In Care Facility' is false, the rule now does not care what the input for 'Facility is an Alcohol Or Drug Care Facility' is - it will simply ignore evaluating that whether it is true, false or null, since the status of the facility is irrelevant if the client is not in a facility.

>
>     * If there is an amount in input "Receiving Diet Supplement" it will reduce the amount of Additional Nutritional Items by the amount of Diet. This doesn't match policy that says "Recipients who are in receipt of the nutritional items portion of the MNS are not eligible for any other nutrition-related supplement". If the intent of this is to be applied only in the first month of eligibility then this setup is correct for One Time payments but not for Monthly/Ongoing.
> 

- We generally haven't addressed the monthly/ongoing vs one time payment distinction for the rules, so if it works for now, we could keep this field present. A future implementation could look to separate this into multiple rules? Technically the current implementation would be an extension of the ongoing monthly functionality, so removing this feature would be simpler than adding it back in the future. As well, we could assume that the frontend would simply send a $0 amount for the diet supplement field in the future, since the client couldn't receive both? Thoughts?

>
>     * Input “Verification of Need” is something that Health Assistance determines as part of adjudication process, and that is not in scope for BRE. MNS should be treated like PWD designation as both are adjudicated by Health Assistance.
> 

- removed the verification of need input.

sbuhovac reviewed (approved):
Validation Testing passed in the current state, as agreed upon in a meeting.
